vector/src/main/java/im/vector/app/features/spaces/explore/SpaceDirectoryFragment.kt

```diff
@@ -77,6 +88,32 @@ class SpaceDirectoryFragment @Inject constructor(
         views.toolbar.title = title
     }
 
+    override fun onPrepareOptionsMenu(menu: Menu) = withState(viewModel) { state ->
+        menu.findItem(R.id.spaceAddRoom)?.let {
+            it.isVisible = state.canAddRooms
+        }
+        menu.findItem(R.id.spaceCreateRoom)?.let {
```

Comment:
no need to use `let` blocks here, you can simply write
`menu.findItem(R.id.spaceAddRoom)?.isVisible = state.canAddRooms`

Reply:
done